source/documentation/internal/issue-labelling.html.md.erb

```diff
@@ -38,7 +46,6 @@ A story type Issue will typically have 4 labels assigned to it:
 | 1Password                  | Needs Refining    | Must Have       | Community        | Ruby      | Project              |
 | Auth0                      | Refined           | Should Have     | Documentation    | Python    | Technical Debt       |
 | CircleCI                   | Agreed for Sprint | Could Have      | EPIC             | Terraform | Unplanned            |
-| DNS-IAC                    | Purged            | Wont Have       | Feature Request  |           | Experimental         |
```

Comment:
Did you mean to remove all the other labels in that row?

Reply:
Ahh I read them as they were related to DNS-IAC, I'll get them added back in!

Reply:
Cheers